### PR TITLE
Add showEditInChartStudio config option

### PR DESCRIPTION
**Pull request by etpinard**
as a supplement to `showSendToCloud` (ref: https://github.com/plotly/plotly.js/pull/3307)

Demo: https://codepen.io/etpinard/pen/YmzbYG

Peek 2019-07-18 15-27

Required for 1.49.0 - cc @plotly/plotly_js @nicolaskruchten

### CODE REVIEW

File: src/fonts/ploticon/ploticon.svg [hunk 1]
```diff
@@ -26,6 +26,7 @@
...
 <glyph glyph-name="selectbox" unicode="&#xe812;" d="m0 850l0-143 143 0 0 143-143 0z m286 0l0-143 143 0 0 143-143 0z m285 0l0-143 143 0 0 143-143 0z m286 0l0-143 143 0 0 143-143 0z m-857-286l0-143 143 0 0 143-143 0z m857 0l0-143 143 0 0 143-143 0z m-857-285l0-143 143 0 0 143-143 0z m857 0l0-143 143 0 0 143-143 0z m-857-286l0-143 143 0 0 143-143 0z m286 0l0-143 143 0 0 143-143 0z m285 0l0-143 143 0 0 143-143 0z m286 0l0-143 143 0 0 143-143 0z" horiz-adv-x="1000" />
 <glyph glyph-name="spikeline" unicode="&#57408;" d="M512 409c0-57-46-104-103-104-57 0-104 47-104 104 0 57 47 103 104 103 57 0 103-46 103-103z m-327-39l92 0 0 92-92 0z m-185 0l92 0 0 92-92 0z m370-186l92 0 0 93-92 0z m0-184l92 0 0 92-92 0z"/>
+<glyph glyph-name="pencil" unicode="&#x270E;" horiz-adv-x="1792" vert-adv-y="1792" d="M491 1536l91-91-235-235-91 91v107h128v128h107zm523-928q0-22-22-22-10 0-17 7l-542 542q-7 7-7 17 0 22 22 22 10 0 17-7l542-542q7-7 7-17zm-54-192l416 416-832 832h-416v-416zm683 96q0 53-37 90l-166 166-416-416 166-165q36-38 90-38 53 0 91 38l235 234q37 39 37 91z"/>
```
Comment (etpinard) [inline src/fonts/ploticon/ploticon.svg]: Taken form 

https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/svg/pencil.svg

**Comment by etpinard** [comment]
Oh and while doing this, I noticed that all the files in `src/fonts/ploticon/` except for `src/fonts/ploticon/ploticon.svg` are not used anymore. I believe they are artifacts from our font-awesome account and/or pre-open-source days.

Moreover, since we don't appear to use our font-awesome account to add new icons (e.g. in https://github.com/plotly/plotly.js/pull/1461 and https://github.com/plotly/plotly.js/pull/3068), I'm thinking of dropping the ploticon build step (mostly in `tasks/util/pull_font_svg.js`) altogether. To add icons we would simply have to modify the `build/ploticon.js` file by hand.

Any objections @plotly/plotly_js ?

**Comment by antoinerg** [comment]
> Any objections @plotly/plotly_js ?

None from me, feel free to :hocho: !

File: src/fonts/ploticon/ploticon.svg (hunk 1 above)
Comment (antoinerg) [reply src/fonts/ploticon/ploticon.svg]: This file will be deleted in #4062

**Comment by antoinerg** [comment]
Thanks, @etpinard for the nice addition! I like the icon: it's very clear.

:dancer:

**Pull request merged**